lms/djangoapps/verify_student/apps.py

```diff
@@ -0,0 +1,19 @@
+"""
+Student Identity Verification Application Configuration
+"""
+
+from django.apps import AppConfig
+
+
+class VerifyStudentConfig(AppConfig):
+    """
+    Application Configuration for verify_student.
+    """
+    name = 'verify_student'
```

Comment:
Agreeing with @jmbowman above - this name wouldn't work unless `lms/djangoapps` was on the `PYTHONPATH`.